### ISSUE TITLE
Login not working. No such signedprekeyrecord!

### ISSUE THREAD

**Issue by nightcore125** [tgalal/yowsup#1549]
Tested with python2.7 and 3.4 and deleted axolotl.db. The error occurs when login.

```
yowsup      v2.4.102

Copyright (c) 2012-2016 Tarek Galal
http://www.openwhatsapp.org

This software is provided free of charge. Copying and redistribution is
encouraged.

If you appreciate this software and you would like to support future
development please consider donating:
http://openwhatsapp.org/yowsup/donate


Yowsup Cli client
==================
Type /help for available commands

[offline]:Auth: Logged in!
[connected]:ERROR:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 11423306
ERROR:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 2265072
ERROR:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 13656587
Traceback (most recent call last):
  File "/usr/local/bin/yowsup-cli", line 368, in <module>
    if not parser.process():
  File "/usr/local/bin/yowsup-cli", line 268, in process
    self.startCmdline()
  File "/usr/local/bin/yowsup-cli", line 297, in startCmdline
    stack.start()
  File "/usr/local/lib/python3.4/dist-packages/yowsup/demos/cli/stack.py", line 25, in start
    self.stack.loop(timeout = 0.5, discrete = 0.5)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/stacks/yowstack.py", line 187, in loop
    asyncore.loop(*args, **kwargs)
  File "/usr/lib/python3.4/asyncore.py", line 208, in loop
    poll_fun(timeout, map)
  File "/usr/lib/python3.4/asyncore.py", line 153, in poll
    read(obj)
  File "/usr/lib/python3.4/asyncore.py", line 87, in read
    obj.handle_error()
  File "/usr/lib/python3.4/asyncore.py", line 83, in read
    obj.handle_read_event()
  File "/usr/lib/python3.4/asyncore.py", line 442, in handle_read_event
    self.handle_read()
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/network/layer.py", line 88, in handle_read
    self.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/network/layer.py", line 96, in receive
    self.toUpper(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/__init__.py", line 75, in toUpper
    self.__upper.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/stanzaregulator/layer.py", line 29, in receive
    self.processReceived()
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/stanzaregulator/layer.py", line 52, in processReceived
    self.processReceived()
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/stanzaregulator/layer.py", line 52, in processReceived
    self.processReceived()
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/stanzaregulator/layer.py", line 49, in processReceived
    self.toUpper(oneMessageData)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/__init__.py", line 75, in toUpper
    self.__upper.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/auth/layer_crypt.py", line 65, in receive
    self.toUpper(payload)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/__init__.py", line 75, in toUpper
    self.__upper.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/coder/layer.py", line 35, in receive
    self.toUpper(node)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/__init__.py", line 75, in toUpper
    self.__upper.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/logger/layer.py", line 14, in receive
    self.toUpper(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/__init__.py", line 75, in toUpper
    self.__upper.receive(data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/layer.py", line 124, in receive
    self.onMessage(protocolTreeNode)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/layer.py", line 146, in onMessage
    self.handleEncMessage(protocolTreeNode)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/layer.py", line 231, in handleEncMessage
    self.handleWhisperMessage(node)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/layer.py", line 277, in handleWhisperMessage
    encMessageProtocolEntity = EncryptedMessageProtocolEntity.fromProtocolTreeNode(node)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/protocolentities/message_encrypted.py", line 53, in fromProtocolTreeNode
    encNode.data.encode('latin-1') if sys.version_info >= (3,0) else encNode.data)
  File "/usr/local/lib/python3.4/dist-packages/yowsup/layers/axolotl/protocolentities/message_encrypted.py", line 24, in setEncProps
    assert encType in "pkmsg", "msg"
AssertionError: msg
```

**Comment by jlguardi** [comment]
Try wa16 or my master (both are being developed). Your issue I due to encrypted group messages which are not supported by master.

**Comment by drmaposa** [comment]
@jlguardi 

I have got the same issue.

I tried to send two simple messages seperatley

"Hello" and "Test"

I get thw following

`Yowsup Cli client
==================
Type /help for available commands

[offline]:/L
Auth: Logged in!
[connected]:WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
Sent: 1461830033-1
[connected]:
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:Too many retries!! Going to send the delivery receipt myself!
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:No such signedprekeyrecord! 31074
WARNING:yowsup.layers.axolotl.layer:Too many retries!! Going to send the delivery receipt myself!
`

I installed the current master by `git clone https://github.com/jlguardi/yowsup.git`

I tried deleting the axolotl.db but cant get the issue resolved, even restarting the linux server wont make a change. Any help will be greatly appreciated
